### PR TITLE
Add selfswab sync to bq task

### PR DESCRIPTION
**Pull request by erikh360**

### CODE REVIEW

File: selfswab/models.py [hunk 1]
```diff
@@ -56,6 +56,32 @@ class SelfSwabScreen(models.Model):
...
+            "id": str(self.id),
+            "contact_id": self.contact_id,
+            "msisdn": self.hashed_msisdn,
+            "age": self.age,
+            "gender": self.gender,
+            "facility": self.facility,
+            "risk_type": self.risk_type,
+            "timestamp": self.timestamp.isoformat(),
+            "occupation": self.occupation,
+            "employee_number": self.employee_number,
```
Comment (rudigiesler) [inline selfswab/models.py]: Would the employee number not be considered the same level of PII of the msisdn?

File: selfswab/models.py [hunk 2]
```diff
@@ -43,7 +44,6 @@ class SelfSwabScreen(models.Model):
     gender = models.CharField(max_length=10, choices=GENDER_CHOICES)
     facility = models.CharField(max_length=255, blank=False)
     risk_type = models.CharField(max_length=10, choices=RISK_TYPES)
-    timestamp = models.DateTimeField(default=timezone.now)
```
Comment (rudigiesler) [inline selfswab/models.py:46]: Do we not need a migration for this?

File: selfswab/models.py (hunk 1 above)
Comment (erikh360) [reply selfswab/models.py]: You're right

File: selfswab/models.py (hunk 2 above)
Comment (erikh360) [reply selfswab/models.py:46]: It was a duplicate field, but I'll run makemigrations just to be sure

Comment (erikh360) [reply selfswab/models.py:46]: `No changes detected`

**Review by rudigiesler — APPROVED** [review]

**Pull request merged**